android/sdl_android/src/androidTest/java/com/smartdevicelink/managers/video/VideoStreamingRangeTests.java

```diff
@@ -0,0 +1,46 @@
+package com.smartdevicelink.managers.video;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import com.smartdevicelink.managers.video.resolution.Resolution;
+import com.smartdevicelink.managers.video.resolution.VideoStreamingRange;
+import com.smartdevicelink.proxy.rpc.ImageResolution;
+
+import junit.framework.TestCase;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class VideoStreamingRangeTests extends TestCase {
+
+    VideoStreamingRange testRange;
+    Resolution minResolution;
+    Resolution maxResolution;
+    Double minimumDiagonal;
+    Double minimumAspectRatio;
+    Double maximumAspectRatio;
+
+    @Before
+    public void setUp() {
+        minResolution = new Resolution(640, 480);
+        maxResolution = new Resolution(1920, 1080);
+        minimumDiagonal = 3.3;
+        minimumAspectRatio = 1.0;
+        maximumAspectRatio = 2.0;
+        testRange = new VideoStreamingRange(minResolution, maxResolution, minimumDiagonal, minimumAspectRatio, maximumAspectRatio);
+    }
+
```

Comment:
@kostyaBoss, Please add more unit tests here,

There should be tests for 
`testResolutionOutOfRange`
`testAspectRatioOutOfRange`
`testResolutionWithNullResolutionRange` (Range resolutions are set to null and should return true)
`testAspectRatioWithNullAspectRatioRange` (Range aspect Ratios are set to null and should return true)
`testResolutionWithDisabledResolutionRange` (Range resolutions have 0 width and height values and should return false)
`testResolutionWithDisabledAspectRatioRange` (Range aspectRatios have 0 values and should return false)